src/App.js

```diff
@@ -1,16 +1,105 @@
 import React from "react";
-import logo from "./logo.png";
+import axios from "axios";
 import "./App.css";
 
+const OPEN_WEATHER_API_KEY = "58b6f653523bf7394917009e6410008c";
```

Comment:
Never commit your API key openly like so. This should only sit in the local .env file, and should never be commited or pushed to github. Please revise https://bootcamp.rocketacademy.co/2-full-stack/2.2-advanced-react/2.2.7-environmental-variables